### PR TITLE
ci: adding devops as codeowners

### PR DESCRIPTION
**Pull request by Hakim187**
this PR is created in order to add the devops team as codeowner

### CODE REVIEW

**Comment by graviteeio** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/gravitee-io/keeper?version=dev:6c404543956fcc05ac06cce4ea1bb4591a97bcf2

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,2 @@
 *             @gravitee-io/techops @jgiovaresco
+*             @gravitee-io/devops
```
Comment (jgiovaresco) [inline .github/CODEOWNERS:2]: Maybe we can only keep one line:

```suggestion
*             @gravitee-io/devops
```

**Comment by graviteeio** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/gravitee-io/keeper?version=dev:1346419feac3c5957d53ad47c20a5b604ead2a31

File: .github/CODEOWNERS (hunk 1 above)
Comment (Hakim187) [reply .github/CODEOWNERS]: done

**Comment by graviteeio** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/gravitee-io/keeper?version=dev:8d10ef54484c84ba31deeca5735251106a023bb2

**Comment by graviteeio** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/gravitee-io/keeper?version=dev:e323ba019e07ae244d701252e301c72e7a10e347

**Pull request merged**